stdlib/builtins.pyi

```diff
@@ -512,49 +515,52 @@ class bytes(ByteString):
     def isspace(self) -> bool: ...
     def istitle(self) -> bool: ...
     def isupper(self) -> bool: ...
-    def join(self, __iterable_of_bytes: Iterable[ByteString | memoryview]) -> bytes: ...
-    def ljust(self, __width: SupportsIndex, __fillchar: bytes = ...) -> bytes: ...
+    def join(self, __iterable_of_bytes: Iterable[ReadableBuffer]) -> bytes: ...
+    def ljust(self, __width: SupportsIndex, __fillchar: bytes | bytearray = ...) -> bytes: ...
```

Comment:
Unfortunately, this will also accept `memoryview` at the moment, but having it more explicit can't hurt.

Reply:
That's a mypy bug :)

Reply:
It's working [as documented](https://docs.python.org/3/library/typing.html?highlight=typing%20bytearray#typing.ByteString). In the past when reviewing I've always asked people to remove `bytearray` from argument types due to that.